content/3_Silver/Func_Graphs.mdx

```diff
@@ -720,6 +720,83 @@ It's also possible to use `floyd(x)` to generate answers for all vertices in the
 containing `x` without using adjacency lists.
 
 <LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+vector<int> res;
+vector<int> arr;
+
+/*
+ *fills up all vertices from curr to first_cycle,
+ *whose answer has already been calculated
```

Comment:
💀 did you just undo my changes

Reply:
Sorry, I didn't saw you there. I already changed it the way you suggested.